metrics_ci_integration/lib/jenkins/model/jenkins_build.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:ci_integration/jenkins/model/jenkins_build_artifact.dart';
+import 'package:ci_integration/jenkins/model/jenkins_building_job.dart';
+import 'package:equatable/equatable.dart';
+
+/// A class representing a single Jenkins build.
+class JenkinsBuild extends Equatable {
+  /// A build number used to identify the build within a [JenkinsBuildingJob].
+  ///
+  /// This is unique only within a job's list of builds.
+  final int number;
+
+  /// A duration of this build.
+  final Duration duration;
+
+  /// A timestamp this build has been started at.
+  final DateTime timestamp;
+
+  /// A result of this build.
+  final String result;
+
+  /// A link to access this build in Jenkins.
+  final String url;
+
+  /// A list of [JenkinsBuildArtifact]s generated during the build.
+  final List<JenkinsBuildArtifact> artifacts;
+
+  @override
+  List<Object> get props =>
+      [number, duration, timestamp, result, url, artifacts];
+
+  const JenkinsBuild({
+    this.number,
+    this.duration,
+    this.timestamp,
+    this.result,
+    this.url,
+    this.artifacts,
+  });
+
+  /// Creates an instance of Jenkins build from the decoded JSON object.
+  ///
+  /// Returns `null` if [json] is `null`.
+  factory JenkinsBuild.fromJson(Map<String, dynamic> json) {
+    if (json == null) return null;
+
+    final timestamp = json['timestamp'] == null
```

Comment:
Maybe we can replace this with int.tryParse() to make it more reliable?

Reply:
I'm not sure I understand what do you mean. 

`int.tryParse()` throws if argument is `null`. Do you mean to change a `timestamp` property type to `int` instead of `DateTime` to just use `json['timestamp'] as int`?

Reply:
@vova-beloded can you please take a look and clarify?

Reply:
I mean replace this check with something like this `int.tryParse('${json['timestamp']}') == null` to be sure that json['timestamp'] is a valid integer.

Reply:
I think it is more Jenkins-side to ensure that `timestamp` is an integer.

Reply:
Ok, then we can leave it as is, I think.